src/devices/Units/tests/SetCultureForTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+
+namespace Units.Tests
+{
+    public sealed class SetCultureForTest : IDisposable
+    {
+        private CultureInfo _previousCulture;
+        private CultureInfo _previousUiCulture;
+
+        public SetCultureForTest(string cultureName)
+        {
+            _previousCulture = CultureInfo.CurrentCulture;
+            _previousUiCulture = CultureInfo.CurrentUICulture;
+            var culture = new CultureInfo(cultureName, false);
+            Thread.CurrentThread.CurrentCulture = culture;
```

Comment:
while these will work for this project because we disabled parallelization I'd prefer we removed these tests or only tested it with explicit culture - this will be a time bomb - will start blowing when we start running tests in parallel (one way to do it reliably is to use RemoteExecutor, i.e.: https://github.com/dotnet/runtime/blob/master/src/libraries/System.Diagnostics.Process/tests/ProcessTestBase.cs#L55)

Reply:
I don't think parallelization will cause problems here, since the culture is per Thread. Even if parallelization is active, no two tests will run simultaneously on the same thread.